Header head
string description
bool attach_images
int8 position
uint8 type
float32 dist_manhole
string subtype
---
bool result
